index.html

```diff
@@ -1565,83 +1612,129 @@
-                tapped, otherwise reject <var>promise</var> with
-                <code>"InvalidAccessError"</code> and abort these steps.
-              </li>
-              <li>
-                If <var>target</var> has the value <code>"peer"</code>,
-                then the sending MAY only happen if an <a>NFC peer</a> is
-                tapped, otherwise reject <var>promise</var> with
-                <code>"InvalidAccessError"</code> and abort these steps.
-              </li>
-              <li>
-                If <var>target</var> has the value <code>"any"</code>
-                (which is the default value), then the sending should happen if
-                either an <a>NFC peer</a> or an <a>NFC tag</a> is tapped within
-                the specified timeout.
+                The <var>timer</var> expires, reject <var>promise</var>
+                with <code>"TimeoutError"</code> and abort these steps.
               </li>
               <li>
-                If the connection is lost during the transfer, or an error is
-                signaled by the underlying platform, reject
-                <var>promise</var> with <code>"<a>NetworkError</a>"</code> and
-                terminate this algorithm.
+                If <var>timer</var> is active, and the
+                <a href="cancelPush"><code>cancelPush()</code></a>
+                method is called, reject <var>promise</var> with
+                <code>"<a>AbortError</a>"</code>, as described in the
+                <a href="#steps-cancelPush"><code>cancelPush()</code>
+                algorithm</a>.
               </li>
               <li>
-                If the transfer is successful, cancel <var>timer</var> and
-                resolve <var>promise</var>.
+                When an <a>NFC device</a> <var>device</var> comes within
+                communication range,
+                <ol>
+                  <li>
+                    If <var>suspended</var> variable
+                    <a href="#adapter-suspended">associated</a> with the current
+                    <a>NFC adapter</a> is <code>true</code>, then
+                     reject <var>promise</var> with
+                    <code>"InvalidAccessError"</code> and abort these steps.
+                  </li>
+                  <li>
+                    If <var>device</var> is an <a>NFC tag</a>, and
+                    <var>tagMessage</var> is <code>null</code>, then  reject
+                    <var>promise</var> with <code>"InvalidAccessError"</code>
+                    and abort these steps.
+                  </li>
+                  <li>
+                    If <var>device</var> is an <a>NFC peer</a>, and
+                    <var>peerMessage</var> is <code>null</code>, then  reject
+                    <var>promise</var> with <code>"InvalidAccessError"</code>
+                    and abort these steps.
+                  </li>
+                  <li>
```

Comment:
In most web specs, the "UA" is specified as if it included the underlying platform, and the spec describes the visible result instead of the interaction with the OS. E.g. http://tools.ietf.org/html/rfc6455#section-4.1 talks about the client opening a TCP connection, even though the underlying platform is actually the entity opening the connection.

However, "initiate" is fine here if you'd rather keep the separation.